src/integrationTest/groovy/wooga/gradle/unity/UnityTaskIntegrationSpec.groovy

```diff
@@ -467,6 +467,7 @@ abstract class UnityTaskIntegrationSpec<T extends UnityTask> extends UnityIntegr
         result.standardOutput.contains(mockUnityStartupMessage)
     }
 
+    // TODO: This test is incorrect since the assumption is wrong?
```

Comment:
This inflicts two different things. One is the flag `-logFile` provided to unity to instruct Unity to either save to file or write to stdout. When we want both! Then we need to instruct to send to stdout and the plugin will write it to the file.

Here a truth table as reference

## Unity >= 2019
| logPath set | logToStdout set | is log written to file? | flag set in unity cli |
| ----------- | --------------- | ----------------------- | --------------------- |
| :yes:       | :no:            | :yes:                   | `-logFile {path}`     |
| :yes:       | :yes:           | :yes:			  | `-logFile -`          |
| :no:        | :yes:           | :no:                    | `-logFile -`          |
| :no:        | :no:            | :no:                    | ``                    |

## Unity <= 2018 
| logPath set | logToStdout set | is log written to file? | flag set in unity cli |
| ----------- | --------------- | ----------------------- | --------------------- |
| :yes:       | :no:            | :yes:                   | `-logFile {path}`     |
| :yes:       | :yes:           | :yes:			  | `-logFile`            |
| :no:        | :yes:           | :no:                    | `-logFile`            |
| :no:        | :no:            | :no:                    | ``                    |

Reply:
Ah sorry;  I didn't mean to commit that comment.

Reply:
I was unsure if we cover this in the tests correctly. What I see is only one part. But not the combination of flag and if the file is written or not and if the stdout has the correct values. I guess we added this before already or?

Reply:
There's a few tests above that check for stdout. would you take a look? 
We could perhaps write another one instead?

Reply:
Though I see now that test isn't function since that property is no longer there.

Reply:
I am not sure if the table is correct for cases 4, 8. 
Even if no log file is set, the plugin still sets one by convention.